config/application.rb

```diff
@@ -17,6 +17,32 @@ class Application < Rails::Application
     config.app_protocol = ENV['APP_PROTOCOL']
     config.base_url = "#{config.app_protocol}://#{config.app_domain_name}/"
 
+    ##
```

Comment:
Great documentation. Just a thought. Would it be of any benefit to have this referred to in the PR description as is really the essence of the accumulative work toward this PR?